File: 905. Sort Array By Parity/solution.java
class Solution {
    public int[] sortArrayByParity(int[] nums) {
        int []arr=new int[nums.length];
        int ii=0,j=nums.length-1;
        for(int i=0;i<nums.length;i++){
            if(nums[i]%2!=0){
                arr[j]=nums[i];
                j--;
            }
            else{
                arr[ii]=nums[i];
                ii++;
            }
        }
        return arr;
    }
}

File: 905. Sort Array By Parity/solution.py
class Solution(object):
    def sortArrayByParity(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        arr=[' ']*len(nums)
        i=0
        j=(len(nums)-1)
        for k in nums:
            if k%2!=0:
                arr[j]=k
                j-=1
            else:
                arr[i]=k
                i+=1
        return arr

